crates/beerus-rpc/example_rpc_calls/stark_getBlockWithTxs.hurl

```diff
@@ -0,0 +1,7 @@
+POST http://0.0.0.0:3030
+Content-Type: application/json
+{
+    "jsonrpc":"2.0","method":"stark_getBlockWithTxs","params":[
```

Comment:
Can we change to `starknet_` to reflect specs as well as file name of the hurl + newline at the bottom